### ISSUE TITLE
Does a similar view possible inside emacs ?

### ISSUE THREAD

**Issue by louisdecharson** [rougier/agenda#5]
Hi 👋 !

First thank you very much for the project !
Do you know / have built a similar view inside Emacs ?

**Comment by rougier** [comment]
Partially. See https://www.reddit.com/r/orgmode/comments/go5jj3/calendar_busy_colors_according_to_agenda/

**Comment by agenbite** [comment]
Wow, it'd be sooo nice to have this inside emacs, especially as part of an org-mode buffer... I've tried calfw but definitely it's not for me.

**Comment by rougier** [comment]
I think I've more experience now and it might be possible. Starting point would be https://github.com/rougier/nano-agenda

**Comment by agenbite** [comment]
Count on me for testing it out! :)

**Comment by agenbite** [comment]
Do you know if there's a way to gather your nano-agenda and a standard org-mode agenda in the same buffer? I mean, having a calendar on top, and the standard agenda below.

**Comment by rougier** [comment]
Yes, nano-agenda can be put on to of the regular agenda but not in the same buffer (because I need to control highligting wuite precisely).

**Comment by agenbite** [comment]
I see. Imho, it'd be a great addition to org-agenda to display a calendar in the same buffer, even if colours are not as meaningful as in your nano-agenda. :)
